util/src/main/java/io/kubernetes/client/util/CallGeneratorParams.java

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.util;
+
+public class CallGeneratorParams {
```

Comment:
I'm not fully convinced this class needs to be separate, but I couldn't get it to compile when it was part of the `Cache` class.

Reply:
java lambda function can have only one parameter as its input, seems that we have to separate them. wdyt?